package.json

```diff
@@ -68,12 +68,13 @@
     "whatwg-fetch": "1.0.0"
   },
   "dependencies": {
-    "babel-polyfill": "6.26.0",
     "@patternfly/react-console": "1.10.22",
+    "babel-polyfill": "6.26.0",
```

Comment:
Does it matter which package goes first?

Reply:
Technically no.  But `yarn add` will sort the entire dependency list alphabetically.